### PR TITLE
Database delete meal

### PR DESCRIPTION
**Pull request by Sindahlof**

### CODE REVIEW

File: CarboPlanner/src/views/ResultScreen.tsx [hunk 1]
```diff
@@ -279,7 +264,7 @@ export default function Result({route}) {
                     </Card>
                     {/*save or cancel*/}
                     <View style={styles.saveOrCancel}>
-                        <TouchableOpacity onPress={() => {handleSaveButtonPress()}} style={styles.saveButton}>
+                        <TouchableOpacity onPress={() => {handleSaveButtonPress(), console.log(createData())}} style={styles.saveButton}>
```
Comment (Nolle10) [inline CarboPlanner/src/views/ResultScreen.tsx:267]: Are we sure, we want to print it in the console? That could be a lot of text...

**Review by Nolle10 — APPROVED** [review]
Looks good, just a single console.log I was wondering about

File: CarboPlanner/src/views/ResultScreen.tsx (hunk 1 above)
Comment (Nolle10) [inline CarboPlanner/src/views/ResultScreen.tsx]:
```suggestion
                        <TouchableOpacity onPress={() => handleSaveButtonPress()} style={styles.saveButton}>
```

**Review by Nolle10 — CHANGES_REQUESTED** [review]

**Review by Nolle10 — APPROVED** [review]
lgtm

File: CarboPlanner/src/views/ResultScreen.tsx [hunk 2]
```diff
@@ -173,32 +164,21 @@ export default function Result({route}) {
...
-                    {/*
-                    <View style={styles.header}>
-                        <TouchableOpacity onPress={() => { navigation.navigate('Home'); }} style={styles.goBackButton} />
-                        <View style={styles.resultHeaderContainer}>
-                            <Text style={styles.resultHeaderText}>Result</Text>
-                        </View>
-                    </View>
-                    */}
                     {route.params.uuidKey ? (
                     <View></View>
```
Comment (frmad) [inline CarboPlanner/src/views/ResultScreen.tsx:178]: Delete
```suggestion
                   
```

File: CarboPlanner/src/views/HomeScreen.tsx [hunk 3]
```diff
@@ -49,70 +50,78 @@ export default function HomeScreen(){
...
+      </TouchableOpacity>
+    </View>
 
       <Card>
         <View style={styles.row}>
           <CircularSlider value={1200} max={2000}/>
         </View>
         <View style={styles.row}>
-            <MacroProgressBar name={"Carbs"} value={50} max={210} />
+            <MacroProgressBar name={"Carbs"} value={0.5} max={210} />
```
Comment (frmad) [inline CarboPlanner/src/views/HomeScreen.tsx]: Changed in another branch
```suggestion
            <MacroProgressBar name={"Carbs"} value={50} max={210} />
```

File: CarboPlanner/src/views/HomeScreen.tsx [hunk 4]
```diff
@@ -49,70 +50,78 @@ export default function HomeScreen(){
...
+                fat={mealInfo.fat}
+              />
+            );
+          })}
         </List>
       );
     };
+    
+    
+
```
Comment (frmad) [inline CarboPlanner/src/views/HomeScreen.tsx:88]: Delete the lines
```suggestion
    
```

File: CarboPlanner/src/utils/Database/DatabaseActions.tsx [hunk 5]
```diff
@@ -9,57 +8,59 @@ let today = new Date();
...
 };
 
-  export {createData, mainDocRef, edit};
+
+const deleteMeal = async (uuidKey) => {
+  const docSnapshot = await getDoc(mainDocRef);
+  const exsistingData = docSnapshot.data().data;
+
+  const updateData = {...exsistingData}
+  delete updateData[uuidKey]
```
Comment (frmad) [inline CarboPlanner/src/utils/Database/DatabaseActions.tsx]: The naming is a bit confusing
```suggestion
  delete meal[uuidKey]
```

**Pull request merged**